posthog/models/cohort.py

```diff
@@ -75,9 +78,12 @@ class Cohort(models.Model):
     deleted: models.BooleanField = models.BooleanField(default=False)
     groups: models.JSONField = models.JSONField(default=list)
     people: models.ManyToManyField = models.ManyToManyField("Person", through="CohortPeople")
+    version: models.IntegerField = models.IntegerField(default=0)
+    pending_version: models.IntegerField = models.IntegerField(default=0)
```

Comment:
Will this lock the table? Should be `null=True`?

Reply:
lock how? setting defaults should be fine

Reply:
This is the SQL from running ./manage.py sqlmigrate posthog 0205

```
BEGIN;
--
-- Add field version to cohort
--
ALTER TABLE "posthog_cohort" ADD COLUMN "version" integer DEFAULT 0 NOT NULL;
ALTER TABLE "posthog_cohort" ALTER COLUMN "version" DROP DEFAULT;
--
-- Add field pending_version to cohort
--
ALTER TABLE "posthog_cohort" ADD COLUMN "pending_version" integer DEFAULT 0 NOT NULL;
ALTER TABLE "posthog_cohort" ALTER COLUMN "pending_version" DROP DEFAULT;
--
-- Add field version to cohortpeople
--
ALTER TABLE "posthog_cohortpeople" ADD COLUMN "version" integer DEFAULT 0 NOT NULL;
ALTER TABLE "posthog_cohortpeople" ALTER COLUMN "version" DROP DEFAULT;
COMMIT;

i.e. it it's going to write the default 0 to each row, then remove the default after this. I believe this could take some time, and the table will be locked.

Reply:
yeah, i see. posthog_cohortpeople is a big table too

Reply:
Fixed, nullable now